main.rb

```diff
@@ -0,0 +1,14 @@
+require_relative "lib/pascals_triangle"
+
+begin
+  print "How many rows would you like to show in Pascal's triangle? "
+  input_number = Integer(gets.chomp)
+rescue
+  puts "Use a whole number please(an integer)"
+  retry
+end
+puts "Pascal's triangle with #{input_number} rows is: "
+pt = PascalsTriangle.new
+(input_number + 1).times { |rows|
+  p pt.to_s(rows)
```

Comment:
@StefanGajic can you try to make formatting prettier, similar to assignment example? At least make it work for smaller triangles up to 10 rows. Something like:

```
       1
     1  1
  1  3  3  1
 1  4  6  4  1
```